### PR TITLE
[7.3] ErrorSuppressionFixer - support PHP 7.3

### PR DESCRIPTION
**Pull request by kubawerlos**
Handles also `SilencedDeprecationErrorFixer` and it's proxy to `ErrorSuppressionFixer`.

### CODE REVIEW

File: src/Fixer/LanguageConstruct/ErrorSuppressionFixer.php [hunk 1]
```diff
@@ -164,8 +164,13 @@ private function isDeprecationErrorCall(Tokens $tokens, $index)
             return false;
         }
 
-        $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $tokens->getNextTokenOfKind($index, [T_STRING, '(']));
+        $endBraceIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $tokens->getNextTokenOfKind($index, [T_STRING, '(']));
 
-        return $tokens[$tokens->getPrevMeaningfulToken($end)]->equals([T_STRING, 'E_USER_DEPRECATED']);
+        $prevIndex = $tokens->getPrevMeaningfulToken($endBraceIndex);
```
Comment (keradus) [inline src/Fixer/LanguageConstruct/ErrorSuppressionFixer.php]: i;m really sad to see this change over and over again :(

File: tests/Fixtures/Integration/misc/PHP7_3.test [hunk 2]
```diff
@@ -11,7 +11,6 @@ PHP 7.3 test.
     "multiline_whitespace_before_semicolons": true,
     "native_function_invocation": {"include": ["get_class"]},
     "php_unit_test_case_static_method_calls": {"call_type": "this"},
-    "silenced_deprecation_error": true,
```
Comment (keradus) [inline tests/Fixtures/Integration/misc/PHP7_3.test]: why removed ?

Comment (keradus) [inline tests/Fixtures/Integration/misc/PHP7_3.test]: :+1:

File: src/Fixer/LanguageConstruct/ErrorSuppressionFixer.php (hunk 1 above)
Comment (kubawerlos) [reply src/Fixer/LanguageConstruct/ErrorSuppressionFixer.php]: yeah :( it's similar to checking for leading `\` in functions/classes

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**